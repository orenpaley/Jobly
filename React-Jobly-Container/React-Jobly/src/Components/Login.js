import LoginForm from "./LoginForm";
import "./Login.css";

function Login({ handleChange, setUser }) {
  return (
    <div className="login-page">
      <LoginForm handleChange={handleChange} setUser={setUser} />
      <div className="filler">
        Don't have an account? <a href="/signup">Register</a>
      </div>
    </div>
  );
}

export default Login;
