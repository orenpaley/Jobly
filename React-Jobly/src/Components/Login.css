
.login-page {
  border-top: 3px black solid;
  padding: 0px;
  background-color:rgb(255, 160, 191);
  display: block;
  width: auto;
  height: 500px;

}
form {
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;
  align-items: center;
  padding: 62px;
}
label {
  letter-spacing: .045rem;
  color: rgb(150, 11, 11);
  font-weight: 600;
  width: 95%;
  font-size: 20px;
  text-align: left;

}

.filler {
  display: fixed;
  padding-top: 12px;
  border-top: 3px black solid;
  bottom: 0px;
  width: 100%;
  height: 500px;
  background-color: red;
}

.input-container {
  /* font-size: 20px;
  display: flex;
  flex-direction: row;
  align-items: center;
  padding-bottom: 36px; */
}

.button {
  margin-top: 35px;
  max-width: 138px;
  font-size: 22px;
}