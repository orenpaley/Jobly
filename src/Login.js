import LoginForm from "./LoginForm";

function Login({ handleChange, token, setToken, user, setUser }) {
  return (
    <div>
      <h1>Login</h1>
      <LoginForm
        handleChange={handleChange}
        token={token}
        setToken={setToken}
        user={user}
        setUser={setUser}
      />
    </div>
  );
}

export default Login;
