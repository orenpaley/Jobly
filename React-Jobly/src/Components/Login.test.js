import React from "react";
import { render } from "@testing-library/react";
import Login from "./Login";

//smoke test
test("App Smokeshot", () => {
  render(<Login />);
});

// snapshot test
it("matches snapshot", function () {
  const { asFragment } = render(<Login />);
  expect(asFragment()).toMatchSnapshot();
});
