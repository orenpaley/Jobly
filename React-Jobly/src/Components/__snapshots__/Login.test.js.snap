// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="login-page"
  >
    <form
      class="row g-3 needs-validation"
      novalidate=""
    >
      <div
        class="col-md-4"
      >
        <label
          class="form-label"
          for="username"
        >
          Username
        </label>
        <input
          class="form-control"
          id="username"
          required=""
          type="text"
        />
        <div
          class="valid-feedback"
        >
          Looks good!
        </div>
      </div>
      <div
        class="col-md-4"
      >
        <label
          class="form-label"
          for="password"
        >
          Password
        </label>
        <input
          class="form-control"
          id="password"
          required=""
          type="text"
        />
        <div
          class="valid-feedback"
        >
          Looks good!
        </div>
      </div>
      <button
        class="button btn btn-info"
      >
        Login
      </button>
    </form>
    <div
      class="filler"
    >
      Don't have an account? 
      <a
        href="/signup"
      >
        Register
      </a>
    </div>
  </div>
</DocumentFragment>
`;
